kokoro/continuous.sh

```diff
@@ -16,17 +34,6 @@ docker kill $(docker ps --all --quiet) || true
 # Sets the integration testing project.
 export JIB_INTEGRATION_TESTING_PROJECT=jib-integration-testing
 
-# Restarting Docker for Mac to get around the certificate expiration issue:
-# b/112707824
-# https://github.com/GoogleContainerTools/jib/issues/730#issuecomment-413603874
-# https://github.com/moby/moby/issues/11534
-# TODO: remove this temporary fix once b/112707824 is permanently fixed.
-if [ "${KOKORO_JOB_CLUSTER}" = "MACOS_EXTERNAL" ]; then
-  osascript -e 'quit app "Docker"'
-  open -a Docker
-  while ! docker info > /dev/null 2>&1; do sleep 1; done
-fi
-
 cd github/jib
 
-./gradlew clean build integrationTest --info --stacktrace
+./gradlew clean build :jib-cli:integrationTest --info --stacktrace
```

Comment:
We probably don't mean to just run `jib-cli` here?
AFAIK presubmit is `jib-core` only, but continuous should be everything.